### PR TITLE
[prometheus-postgre-exporter] Add extraEnvs to deployment

### PR DESCRIPTION
**Pull request by JuanRamino**
#### What this PR does / why we need it

I need to pass extra environments to postgres-exporter container.

#### Which issue this PR fixes

*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*

- fixes #

#### Checklist

- [ ] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [ ] Chart Version bumped
- [ ] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Review by monotek — CHANGES_REQUESTED** [review]
please raise chart version.

postgres image could be updated too v0.11.1 too.
see: https://quay.io/repository/prometheuscommunity/postgres-exporter?tab=tags&tag=latest

**Comment by JuanRamino** [comment]
Verision bumped

**Review by monotek — APPROVED** [review]

**Review by monotek — COMMENTED** [review]
please sign your commits to get dco check pass

**Review by monotek — CHANGES_REQUESTED** [review]
please sign your commits to get dco check pass

File: charts/prometheus-postgres-exporter/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
-appVersion: "0.10.1"
+appVersion: "0.11.1"
 description: A Helm chart for prometheus postgres-exporter
 name: prometheus-postgres-exporter
-version: 3.1.5
+version: 3.1.6
```
Comment (monotek) [inline charts/prometheus-postgres-exporter/Chart.yaml:5]: new featuer needs to raise minor version.

```suggestion
version: 3.2.0
```

**Review by monotek — CHANGES_REQUESTED** [review]

**Pull request merged**